<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>howtouse</class>
 <widget class="QDialog" name="howtouse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="delete_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="deleteLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>15</y>
     <width>500</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save2History">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>145</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="save2HistoryLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>500</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="settingsLable">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>500</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="deleteFromHistoryLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>215</y>
     <width>500</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteFromHistory">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
